Doppler.Currency/Program.cs

```diff
@@ -15,6 +16,11 @@ public static void Main(string[] args)
             Host.CreateDefaultBuilder(args)
                 .UseSerilog((hostContext, loggerConfiguration) => 
                     loggerConfiguration.ReadFrom.Configuration(hostContext.Configuration))
+                .ConfigureAppConfiguration(configHost =>
+                {
+                    configHost.AddJsonFile("/run/secrets/appsettings.Secret.json", true);
+                    configHost.AddKeyPerFile("/run/secrets", true);
+                })
```

Comment:
https://github.com/FromDoppler/doppler-currency/pull/16/files#r408865701

> This is out of the scope of this PR. It is fine add it, but would better do it in another PR.

But without these lines, the feature "Make optional Slak Hook" should not be work fine when the deploy hasn't the secret key, let me know if we don't agree with me

Reply:
Yes, it would be nice having these changes in a different PR.

But I am fine if you prefeer keep it here.

Reply:
I created this pr https://github.com/FromDoppler/doppler-currency/pull/18